README.md

```diff
@@ -1,113 +1,93 @@
 
-Firebase Genkit (beta) is a framework with powerful tooling to help app developers build, test, deploy, and monitor AI-powered features with confidence. Genkit is also built from the ground up to be cloud optimized and code-centric, designed to feel familiar to most developers with less new concepts to learn. The Genkit framework itself is free and [open source](./LICENSE), and integrates with many services that have free tiers to get started.
+Firebase Genkit is a framework for TypeScript and Go, designed to help you build and iterate on AI-powered applications and features. It provides open source libraries and integrated tooling to support your AI app development.
 
-Genkit is available for [TypeScript (Node.js)](https://www.npmjs.com/package/genkit), with [Go support](https://github.com/firebase/genkit/tree/main/go) in active development.
+## Who is Genkit for?
 
-To be notified when Genkit for Go is ready for early community testing and get invited to our testers chat, [submit this form](https://docs.google.com/forms/d/e/1FAIpQLSeMYYFlH7bpsTZLWYDFSrgUG-64gkB3w_zXuF7NdTrZG04BLQ/viewform?resourcekey=0-eu_VfD8prb0igHZY5-oQKQ).
+Genkit is for any developer who wants to integrate generative AI capabilities into their apps using TypeScript or Go. Code written with Genkit libraries can be deployed anywhere that supports Node.js and Go runtimes, and the framework supports any model and vector store provider.
 
-> [!NOTE]
-> Since Genkit is currently in beta, this means that the public API and framework design may change in backward-incompatible ways.
-
-Getting started is easy:
-
-Install Genkit:
-
-```bash
-npm i -g genkit
-```
-
-Review the [documentation](https://firebase.google.com/docs/genkit) for details and samples.
-
-## Key features
+Genkit is built by Firebase, Google's app development platform, but does not require you to be a Firebase or Google Cloud user.
 
-### GenAI models
+## Why use Genkit?
 
-**Unified API for generation** across AI models built by Google (Gemini, Gemma) and third party providers. Supports multimodal input, multimedia content generation, and custom options.
+Integrating generative AI into apps is complicated. You may need to tie together various components like generative AI models (LLMs), prompts, vector stores, embedders, and tools - each with provider-specific implementations and quirks. The non-deterministic nature of generative AI models also makes refining AI apps for production challenging.
 
-**Generate structured output** as strongly-typed objects with custom schemas for easy integration into your app.
+Genkit libraries provide lightweight abstractions and unified APIs that make it easy to define and compose AI app components from any provider into cohesive workflows with built-in type-safety and observability. Genkit's integrated tooling accelerates your iteration speed by helping you quickly test, visualize, and debug your app's AI logic.
```

Comment:
Maybe instead of AI logic, "AI workflow and output"?

Reply:
Might be worth calling out specifically flow logic and prompts/model settings